lib/beaker_puppet_helpers/install_utils.rb

```diff
@@ -40,15 +34,12 @@ class InstallUtils
     #   available version. Can also be openvox7, puppet8 and others.
     #   Method is called from beaker_install_helpers
     #
-    # @param [Boolean] nightly
-    #   Whether to install nightly or release packages
-    #
     # @note This method only works on redhat-like and debian-like hosts. There
     #   are no official Puppet releases for other platforms.
     #
-    def self.install_puppet_release_repo_on(host, collection = 'puppet', nightly: false)
+    def self.install_puppet_release_repo_on(host, collection = 'puppet')
       requirement_name = collection.gsub(/\d+/, '')
-      repos = REPOS[requirement_name.to_sym][nightly ? :nightly : :release]
+      repos = REPOS[requirement_name.to_sym][:release]
```

Comment:
Main argument for removing the release layer is that it doesn't add anything and if `requirement_name.to_sym` isn't found, it ends up being `nil[:release]` which crashes.

I didn't spot this in the previous PR, but a safer alternative (if you want to keep the `:release` layer)
```suggestion
      repos = REPOS.dig(requirement_name.to_sym, :release)
```